rpcs3/Emu/IdManager.h

```diff
@@ -141,14 +141,35 @@ namespace id_manager
 	template <typename T>
 	struct id_map
 	{
-		std::vector<std::pair<id_key, std::shared_ptr<void>>> vec{};
+		std::vector<std::pair<id_key, std::shared_ptr<void>>> vec{}, private_copy{};
```

Comment:
Why it's not a local variable?

Reply:
So it will be available in both calls without fetching it twice.